### PR TITLE
Sherlock-80: Documentation Cleanups

### PR DESCRIPTION
**Pull request by MikeHathaway**
# Description of change
## High level
* Cleanup code comments to match the latest specification of the contract. No source code changes.
* https://github.com/sherlock-audit/2023-04-ajna-judging/issues/80

### CODE REVIEW

**Review by grandizzy — APPROVED** [review]
LGTM

**Review by ith-harvey — APPROVED** [review]
LGTM

**Review by prat-gpt — APPROVED** [review]
LGTM

File: src/grants/GrantFund.sol [hunk 1]
```diff
@@ -307,8 +307,8 @@ contract GrantFund is IGrantFund, Storage, ReentrancyGuard {
 
         DistributionPeriod storage currentDistribution = _distributions[_currentDistributionId];
 
-        // cannot add new proposal after end of screening period
-        // screening period ends 72000 blocks before end of distribution period, ~ 80 days.
+        // cannot add new proposal after the screening period ends
+        // screening period ends 525_600 blocks after the start of the distribution period, ~73 days.
```
Comment (EdNoepel) [inline src/grants/GrantFund.sol:311]: Correct time span for 12s blocks.

**Review by EdNoepel — APPROVED** [review]
LGTM

**Pull request merged**